corehq/apps/toggle_ui/views.py

```diff
@@ -146,11 +146,11 @@ def static_toggle(self):
         """
         return find_static_toggle(self.toggle_slug)
 
-    def get_toggle(self, skip_cache=False):
+    def get_toggle(self):
         if not self.static_toggle:
             raise Http404()
         try:
-            return Toggle.get(self.toggle_slug, skip_cache=skip_cache)
+            return Toggle.get(self.toggle_slug)
```

Comment:
so earlier we were using cached response in get but actually fetching the doc when doing post request.
That definitely looks risky. 
I believe now we are doing a fetch both times, which is good.